src-java/network-topology/network-storm-topology/src/main/java/org/openkilda/wfm/topology/network/controller/sw/SwitchFsm.java

```diff
@@ -219,8 +219,8 @@ public void handlePortLinkStateChange(SwitchFsmState from, SwitchFsmState to, Sw
                                           SwitchFsmContext context) {
         AbstractPort port = portByNumber.get(context.getPortNumber());
         if (port == null) {
-            log.error("Port {} is not listed into {}", context.getPortNumber(), switchId);
-            return;
+            log.info("Port {} is not listed into {}, creating new record.", context.getPortNumber(), switchId);
+            port = makePortRecord(Endpoint.of(switchId, context.getPortNumber()));
```

Comment:
is not enough

You must call `portAdd(port, context)` and `updateOnlineStatus(port, context, OnlineStatus.ONLINE)` - take a look on `handlePortAdd` logic.